### PR TITLE
MII jrf - improve wallet password handling

### PR DESCRIPTION
**Pull request by tbarnes-us**
Don't include a JRF OPSS wallet passphrase in the WDT 'createDomain.sh' command line since createDomain.sh echos its command line to stdout.

### CODE REVIEW

**Review by jshum2479 — APPROVED** [review]

File: operator/src/main/resources/scripts/modelInImage.sh [hunk 1]
```diff
@@ -674,16 +674,46 @@ function wdtCreatePrimordialDomain() {
...
+
+  if [ -z "${OPSS_FLAGS}" ]; then
+
+    # We get here for WLS domains, and for the JRF 'first time' case
+
+    # JRF wallet generation note:
+    #  If this is JRF, the unset OPSS_FLAGS indicates no wallet file was specified
+    #  via spec.configuration.opss.walletFileSecret and so we assume that
+    #  this is the first time this domain started for this RCU database.
+    #  We also assume that 'createDomain.sh' will setup new RCU tables for the domain
```
Comment (jshum2479) [inline operator/src/main/resources/scripts/modelInImage.sh:687]: It should be create_rcu_schema.sh

**Review by jshum2479 — APPROVED** [review]

**Review by rjeberhard — APPROVED** [review]

**Comment by rjeberhard** [comment]
@tbarnes-us and @jshum2479, I'm not sure how to resolve this conflict.

**Comment by tbarnes-us** [comment]
> @tbarnes-us and @jshum2479, I'm not sure how to resolve this conflict.

What's the conflict?

**Comment by rjeberhard** [comment]
Click the button that says "Resolve conflicts" and see :). I may have communicated the wrong tone with my statement that I "didn't know how." It looks like two changes that are just next to each other, so the tool complains, but I'm not sure so I wanted you two to weigh in.

**Comment by tbarnes-us** [comment]
> Click the button that says "Resolve conflicts" and see :). I may have communicated the wrong tone with my statement that I "didn't know how." It looks like two changes that are just next to each other, so the tool complains, but I'm not sure so I wanted you two to weigh in.

I think the only difference is that Johnny's pull adds detailed log messages.   You can resolve the problem by including the entirety of my change within the conflict, and then adding the following before my change:

```
  if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
    if [ -z "${OPSS_FLAGS}" ] ; then
      trace INFO "An OPSS wallet was not supplied for the Model in Image JRF domain in its " \
        "'spec.configuration.opss.walletFileSecret' attribute; therefore, it's assumed that this is the first time " \
        "the OPSS RCU database is being accessed by the domain, so a schema and a wallet file will be created. " \
        "Consult the Model in Image documentation for instructions about preserving the OPSS wallet file."
    else
      trace "Creating JRF Primordial Domain"
    fi
  fi
```

**Pull request merged**